dist/alpine.js

```diff
@@ -486,7 +481,9 @@
   }
 
   function lookAheadForMatchingKeyedElementAndMoveItIfFound(nextEl, currentKey) {
-    // If the the key's DO match, no need to look ahead.
+    // We don't know if nextEl is avtually an element so we check that it's not a falsy value first
```

Comment:
// We don't know if nextEl is ~~avtually~~ actually an element so we check that it's not a falsy value first